pkg/plugins/builtin/email/email.go

```diff
@@ -0,0 +1,107 @@
+package email
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"net/smtp"
+	"strings"
+	"text/template"
+
+	"github.com/ovh/utask/pkg/plugins/taskplugin"
+)
+
+// the email plugin send email
+var (
+	Plugin = taskplugin.New("email", "0.1", exec,
+		taskplugin.WithConfig(validConfig, Config{}),
+	)
+)
+
+// Config is the configuration needed to send an email
+type Config struct {
+	SMTPUsername string   `json:"smtp_username"`
+	SMTPPassword string   `json:"smtp_password"`
+	SMTPPort     uint     `json:"smtp_port,omitempty"`
+	SMTPHostname string   `json:"smtp_hostname"`
+	From         string   `json:"from"`
+	To           []string `json:"to"`
+	Subject      string   `json:"subject"`
+	Body         string   `json:"body"`
+}
+
+const emailTemplate = `From: {{.From}}<br />
+To: {{.To}}<br />
+Subject: {{.Subject}}<br />
```

Comment:
Beware, email headers should not contain non-ascii characters, so subject can't contains UTF-8 characters and should be encoded as described in RFC 1342

https://tools.ietf.org/html/rfc1342

Reply:
Thanks for the information, fixed.